translations/de.json

```
@@ -0,0 +1,82 @@
+{
+  "_header": "*******************",
+  "_section": "Display Categories",
+  "_header": "*******************",
+    "solar": "Solar",
+    "battery": "Powerwall",
+    "grid": "Stromnetz",
+    "house": "Haus",
+    "car": "Auto lädt",
+
+  "_header": "*******************",
+  "_section": "General UI terms",
+  "_header": "*******************",
+    "yesterday": "Gestern",
+    "today": "Heute",
+    "unknown": "unbekannte Darstellung!",
+
+  "_header": "*******************",
+  "_section": "Car status",
+  "_header": "*******************",
+    "charging_at": "{NAME} lädt auf mit",
+    "charging_at_plural": "{NAME} und {NUM} laden auf mit",
+    "consuming": "{NAME} is consuming",
+    "consuming_plural": "{NAME} und {NUM} verbrauchen",
+
+    "driving": "{NAME} fährt {LOCATION}",
+    "parked": "{NAME} parkt {LOCATION}",
+    "charging": "{NAME} lädt auf {LOCATION}",
+    "not_charging": "{NAME} ist hier {LOCATION} und lädt nicht",
+      "_comment": "...wobei {LOCATION} ist entweder:",
+        "at_home": "zuhause",
+        "elsewhere": "in {TOWN}",
+
+    "more_than_day": "Abgeschlossen in {DAYS}{LISTSEP}{HOURS}",
+    "less_than_day": "Abgeschlossen in {HOURS}{LISTSEP}{MINUTES}",
```

```suggestion
    "completion_time": "Abgeschlossen in {DAYS}{LISTSEP1}{HOURS}{LISTSEP2}{MINUTES}",
```
Sorry, I changed this one out from under you.